stable/nginx/templates/rbac.yaml

```diff
@@ -0,0 +1,45 @@
+{{- if .Values.rbac.enabled }}
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: {{ template "nginx.fullname" . }}
+---
+kind: Role 
+{{- if semverCompare "^1.8-0" .Capabilities.KubeVersion.GitVersion }}
+apiVersion: rbac.authorization.k8s.io/v1
+{{- else }}
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
`rbac.authorization.k8s.io/v1` is available from 1.8. No need to make this configurable.

Reply:
Heh... copy/pasted that bit from other stable charts. :smiley: